pages/dashboard/index.tsx

```diff
@@ -32,6 +47,7 @@ const ServerSelect: NextPage = () => {
       >
         {/* <Govcrumb /> */}
         <Flex justifyContent='center' alignItems='center'>
+          <Button colorScheme='green' onClick={createCommunity}>Create Community</Button>
```

Comment:
For now, we're gonna leave this as an unpublished route that we navigate to manually

Reply:
Will remove the button.